crate_universe/src/metadata.rs

```diff
@@ -76,7 +76,7 @@ impl MetadataGenerator for Generator {
 }
 
 /// A configuration desrcibing how to invoke [cargo update](https://doc.rust-lang.org/cargo/commands/cargo-update.html).
-#[derive(Debug, PartialEq, Eq)]
+#[derive(Debug, Clone, PartialEq, Eq)]
```

Comment:
This seems related to https://github.com/clap-rs/clap/issues/4286 but is not a big deal for our uses here.